### PR TITLE
redirect requests to third_contact#show

### PR DESCRIPTION
**Pull request by ericcf**

### CODE REVIEW

File: app/assets/javascripts/form/required_fields.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
         // Add an asterisk to the labels of required fields.
         $('input[required="required"], select[required="required"], textarea[required="required"]')
             .closest(".form-group")
-            .children("label")
+            .find("label")
```
Comment (michaelwehrley) [inline app/assets/javascripts/form/required_fields.js]: Did you have problems finding labels b/c it is nested further down then one level?

**Comment by michaelwehrley** [comment]
:eyeglasses: great!

File: app/assets/javascripts/form/required_fields.js (hunk 1 above)
Comment (ericcf) [inline app/assets/javascripts/form/required_fields.js]: Yes. I added a note to that commit, but it's not visible at the PR level.

Comment (michaelwehrley) [inline app/assets/javascripts/form/required_fields.js]: Ahh, another good reason to look closely at the commit message :wink:

**Pull request merged**